test/katex-spec.js

```diff
@@ -1116,6 +1116,31 @@ describe("A markup generator", function() {
     });
 });
 
+describe("A parse tree generator", function() {
+    it("generates a tree", function() {
+        var tree = katex.__parse("\\sigma^2");
+        expect(JSON.stringify(tree)).toEqual(JSON.stringify([
```

Comment:
I think that `toEqual` just does an `===` compare. You could try `expect(_.isEqual(tree, ...)).toBe(true)`?